src/index.html

```diff
@@ -1,12 +1,360 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,300&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png"
+        alt="Moyo logo"
+        class="logo__image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active"> APPLE </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link"> SAMSUNG </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link"> SMARTPHONES </a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#laptopscomputers"
+            class="nav__link"
+            data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link"> GADGETS </a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link"> TABLETS </a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link"> PHOTO </a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link"> VIDEO </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+      <div
+      class="card"
+      data-qa="card">
```

Comment:
Need to fix everywhere
```suggestion
      <div class="card" data-qa="card">
```